packages/queries/src/query.test.ts

```diff
@@ -148,6 +131,7 @@ describe('queries', () => {
                 first_name: string;
                 last_name: string;
             }>;
+            filmActor: {}[];
```

Comment:
this shouldn't be there? were you getting a type error?

Reply:
queries with an empty select should not show up in the query result

Reply:
Nope. I just saw it was missing and added it.

Reply:
> queries with an empty select should not show up in the query result

Ahh, I see now. That suggests we have a bug then